python-package/lightgbm/plotting.py

```diff
@@ -268,9 +268,9 @@ def add(root, parent=None, decision=None):
                 if info in {'split_gain', 'internal_value', 'internal_count'}:
                     label += '\n' + info + ':' + str(root[info])
             graph.node(name, label=label)
-            if root['decision_type'] == 'no_greater':
+            if root['decision_type'] == '<=':
                 l_dec, r_dec = '<=', '>'
-            elif root['decision_type'] == 'is':
+            elif root['decision_type'] == '=':
```

Comment:
I think it is `==` . https://github.com/Microsoft/LightGBM/blob/master/src/io/tree.cpp#L340